src/tests/ftest/io/ior_intercept_verify_data_integrity.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+import threading
+import write_host_file
+from ior_test_base import IorTestBase
+from ior_utils import IorCommand
+from command_utils import CommandFailure
+
+
+class IorInterceptVerifyDataIntegrity(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with mix of dfuse and
+       interception library on a multi server and multi client
+       settings and verify read/write.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        super(IorInterceptVerifyDataIntegrity, self).setUp()
+        # Following line can be removed once the constraint in the
+        # IorTestBase is resolved. #DAOS-3320
+        self.hostlist_clients = self.params.get(
+            "test_clients", "/run/hosts/*")
```

Comment:
Lines 47-48 not needed. The resultant outcome will still be same without these lines. Refer to apricots test.py

Reply:
This test class inherits IorTestBase and in it's set up the hostlist_clients are limited to 1 if the api is POSIX and to override that I've introduced these lines. Otherwise ior will run only with 1 client.